js_modules/dagster-ui/packages/ui-components/src/components/TextInput.tsx

```diff
@@ -100,17 +103,16 @@ const RightContainer = styled.div`
 export const TextInputStyles = css`
   background-color: ${colorBackgroundDefault()};
   border: none;
+  outline: none;
   border-radius: 8px;
-  box-shadow:
-    ${colorBorderDefault()} inset 0px 0px 0px 1px,
-    ${colorKeylineDefault()} inset 2px 2px 1.5px;
   color: ${colorTextDefault()};
   flex-grow: 1;
   font-size: 14px;
   line-height: 20px;
   padding: 6px 6px 6px 12px;
+  box-shadow: none;
   margin: 0;
-  transition: box-shadow 150ms;
+  transition: border 150ms;
```

Comment:
I don't think this is correct -- aren't we using `box-shadow` instead of `border`?